x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/test_now_mode/test_now_mode_flyout.tsx

```diff
@@ -65,7 +65,7 @@ export function TestNowModeFlyout({
         </EuiTitle>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
-        <EuiErrorBoundary>
+        <KibanaErrorBoundary>
```

Comment:
When wrapping our application with `KibanaErrorBoundaryProvider`, we already wrap the page with `KibanaErrorBoundary` out of the box, so removing this will show the error at the page level. Would that be enough in this case, or is there a preference for containing the error at this level? If we don't expect to throw an error here, and it only happens when there is a bug, maybe relying on the page-level `KibanaErrorBoundary` makes more sense.

If the preference is the latter, using `KibanaSectionErrorBoundary` instead of `KibanaErrorBoundary` is more suitable.
https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/shared-ux/error_boundary/README.mdx

Reply:
I think the main idea behind having a separate boundary inside this flyout is that it will preserve the flyout UI, to highlight where in the UI the user is experiencing the problem. Agree we can switch to the section boundary here.

Reply:
<img width="1442" alt="image" src="https://github.com/user-attachments/assets/c79c4689-a6a5-46d7-8f0c-3cd9846c73d9" />

Here's how it looks with the section error boundary. I think that does a good job of expressing that the flyout's content (which is one of the trickier parts of Synthetics and could reasonably have unforeseen issues crop up) had an issue, while not blowing up the user's whole page.